Dayanand Viveki/HTML/Assignment5/biodata.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="assets/css/style.css" />
+        <title>Learn HTML,CSS extrenal stylesheet</title>
+    </head>
+    <body>
+        <div class="main-block">
+            <p class="name">Dayanand Viveki</p>
+            <div class="aboutme">
+                <span class="headline">
+                    <p>About Me</p>
+                </span>
+                <span class="info">
+                    <p>
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Sunt neque praesentium dolorem deserunt quas! Deleniti voluptatibus ea,
+                        suscipit nulla quod unde odit, qui eum provident illum ab, facilis cupiditate vero similique voluptates? Ut aperiam at quo aliquid non
+                        temporibus numquam, mollitia quam error accusamus deleniti eligendi quisquam explicabo doloremque libero veniam voluptate, est impedit
+                        dolore. Quia temporibus quod inventore praesentium sapiente, eligendi, necessitatibus distinctio accusamus perferendis odit, explicabo
+                        dolorem quis ad qui asperiores facere dolore harum maiores labore deserunt eveniet ipsa. Rerum voluptas adipisci velit explicabo sunt
+                        quae et sed! Libero voluptatibus dicta officia iure dolorem. Expedita eaque sapiente magni aliquam eligendi ad perspiciatis adipisci
+                        voluptatum alias maxime quasi numquam facilis cum suscipit, sed fugiat iusto minima neque. Quam architecto, voluptates adipisci
+                        laudantium sit autem earum non a exercitationem veniam dolorum, ullam ratione temporibus eligendi doloribus. Soluta est voluptates
+                        explicabo a rem quos sit error qui alias quibusdam fugit excepturi, perferendis nesciunt unde, omnis in officiis! Aperiam culpa
+                        similique explicabo quos neque numquam possimus praesentium? Soluta vel nam voluptatibus nihil vero facilis quia molestiae, sit quasi
+                        vitae ipsa sint, adipisci nobis commodi aut! Illo suscipit esse dolorem impedit aut praesentium.
+                    </p>
+                </span>
+                <span class="headline">
```

Comment:
remove span give style to p tag only

Reply:
It is easy to give margine both the para through span tag